smac/optimizer/ei_optimization.py

```diff
@@ -285,19 +271,31 @@ def _one_iter(
             all_neighbors = get_one_exchange_neighbourhood(
                 incumbent, seed=self.rng.randint(MAXINT))
 
+            neighbors = []
             for neighbor in all_neighbors:
                 s_time = time.time()
                 acq_val = self.acquisition_function([neighbor], *args)
                 neighbors_looked_at += 1
                 time_n.append(time.time() - s_time)
 
+                if acq_val == acq_val_incumbent:
```

Comment:
Do we really expect exactly the same acq value? Shouldn't this rather have a precision that defines what is means to be the same value?

Reply:
Also, what happens if epsilon is > 0. Don't we miss some points here?

Reply:
I think equal is the definition of a plateau here @mlindauer ? Also, maybe want to drop the epsilon?

Reply:
To be consistent, we should either drop epsilon or also consider the epsilon for our plateau walks.